lua/weapons/cfc_cinder_block/shared.lua

```diff
@@ -0,0 +1,79 @@
+AddCSLuaFile()
+
+DEFINE_BASECLASS( "cfc_simple_base_throwing" )
+SWEP.Base = "cfc_simple_base_throwing"
+
+-- UI stuff
+
+SWEP.PrintName = "Cinder Block"
+SWEP.Category = "CFC"
+SWEP.UseHands = true
+
+if CLIENT then -- killicon, HUD icon and language 'translation'
+    CFCPvPWeapons.CL_SetupSwep( SWEP, "cfc_cinder_block", "materials/vgui/hud/cfc_cinder_block.png" )
+end
+
+SWEP.Slot = 1
+SWEP.ViewModel = Model( "models/weapons/c_grenade.mdl" )
+SWEP.Spawnable = true
+SWEP.Purpose = "Throw bricks at people!"
+SWEP.Instructions = "Heavy object, get a running start to make it hurt more!"
+
+SWEP.IdleHoldType = "slam"
+SWEP.ThrowingHoldType = "melee"
+
+SWEP.ProjectileClass = "cfc_cinderblock_projectile"
+SWEP.InfiniteAmmo = true
+
+SWEP.ModelScale = 1
+SWEP.ModelMaterial = nil
+SWEP.ThrowVelMul = 0.25
+
+SWEP.WorldModel = "models/props_debris/concrete_cynderblock001.mdl"
+SWEP.OffsetWorldModel = true
+SWEP.WMPosOffset = Vector( 8, 2, 0 )
+SWEP.WMAngOffset = Angle( 0, 180, -90 )
+
+SWEP.HeldModel = SWEP.WorldModel
+SWEP.HeldModelPosOffset = Vector( -22, 8, -22 )
+SWEP.HeldModelAngOffset = Angle( 0, -80, 0 )
+
+SWEP.Primary.ThrowAct = { ACT_VM_PULLBACK_HIGH, ACT_VM_THROW }
+SWEP.Primary.LobAct = { ACT_VM_PULLBACK_LOW, ACT_VM_HAULBACK }
+SWEP.Primary.RollAct = { ACT_VM_PULLBACK_LOW, ACT_VM_SECONDARYATTACK }
+
+SWEP.CFC_FirstTimeHints = {
+    {
+        Message = "Get a running start! Faster cinderblocks hurt more!",
+        Sound = "ambient/water/drip1.wav",
+        Duration = 6,
+        DelayNext = 10,
+    },
+    {
+        Message = "Or drop a cinderblock from a building!",
+        Sound = "ambient/water/drip1.wav",
+        Duration = 6,
+        DelayNext = 10,
+    },
+    {
+        Message = "A real cinderblock master never leaves their trusty roof!",
+        Sound = "ambient/water/drip1.wav",
+        Duration = 6,
+        DelayNext = 0,
+    },
```

Comment:
Too much, also it's best to play different drip sounds for each hint
```suggestion
        Sound = "ambient/water/drip2.wav",
        Duration = 5,
        DelayNext = 0,
    },
```